lib/motion/configuration.rb

```diff
@@ -101,5 +99,17 @@ def initialize
     # This is included for completeness. It is not currently used internally by
     # Motion, but it might be required for building view helpers in the future.
     option(:motion_attribute) { "data-motion" }
+
+    def paths
+      @paths ||= begin
+        paths = Rails.application.config.paths.dup
+
+        paths.add "bin", glob: "*"
+
+        paths.add "Gemfile.lock"
+
+        paths
```

Comment:
Nitpick: I think this would look better with `tap`:

```ruby
Rails.application.config.paths.dup.tap do |paths|
   paths.add "bin", glob: "*"
   paths.add "Gemfile.lock"
end
```